numpydoc/numpydoc.py

```diff
@@ -132,6 +132,7 @@ def mangle_docstrings(app, what, name, obj, options, lines):
            app.config.numpydoc_show_inherited_class_members,
            'class_members_toctree': app.config.numpydoc_class_members_toctree}
 
+    cfg.update(options)
```

Comment:
Pass the options on to `get_doc_object`

Reply:
To build the SciPy doc I had to change this to `cfg.update(options or {})` because in [scipyoptdoc](https://github.com/scipy/scipy/blob/master/doc/source/scipyoptdoc.py#L147-L148) we pass `None` for options

Reply:
adopting